ui/lib/core/addon/templates/components/replication-mode-summary.hbs

```diff
@@ -63,15 +63,23 @@
         {{#if (and (eq this.mode "performance") (not (has-feature "Performance Replication")))}}
           <p>
             Performance Replication is a feature of Vault Enterprise Premium.
+            <ExternalLink
+              @href="https://hashicorp.com/products/vault/trial?source=vaultui_Performance%20Replication"
+              class="link"
+              data-test-upgrade-link-performance
+            >
+              Learn more about Performance Replication.
+            </ExternalLink>
           </p>
-          <p class="has-text-centered">
+        {{else if (and (eq this.mode "dr") (not (has-feature "DR Replication")))}}
+          <p>
+            Disaster Recovery is a feature of Vault Enterprise Premium.
             <ExternalLink
               @href="https://hashicorp.com/products/vault/trial?source=vaultui_Performance%20Replication"
```

Comment:
the href links are the same for both because they both lead to a top level replication license doc so it works in both cases. My guess is the old links with anchors are being ignored and sent to this higher level link.